Example configuration.yaml entry (#8)
hassio-addons/addon-mqtt: README.md

[![Docker Layers][layers-shield]][microbadger]
[![Docker Pulls][pulls-shield]][dockerhub]

> ## Configuration Home assistant

Reviewer: ```suggestion
## Home Assistant configuration example
```

> **Note**: _Remember to restart the add-on when the configuration is changed._

Reviewer: ```suggestion
### Notes
_Remember to restart the add-on when the configuration is changed._

_If you are moving from the official add-on to this one, make sure that you change the `broker:` in your configuration from `core-mosquitto` to `a0d7b954-mqtt`._

```

> **Note2**: _When changing from the official add-on to this one, make sure to change configuration.yaml broker ip. "broker: core-mosquitto" to "broker: localhost"_

Reviewer: ```suggestion
```

Author: Didnt remove the entire note, just the last part since that should be obvious.

> ```json

Reviewer: ```suggestion
```yaml
```

# Example configuration.yaml entry
mqtt:

>   broker: localhost

Reviewer: ```suggestion
  broker: a0d7b954-mqtt
```

Reviewer 2: as a newbie who is trying to connect this addon with HA. What's the meaning of `a0d7b954`  ?

Reviewer 3: It's just part of the name to address the add-on: https://developers.home-assistant.io/docs/en/hassio_addon_communication.html#network

  username: !secret mqtt_username
  password: !secret mqtt_password
  client_id: home-assistant
```

> ## Configuration add-on

Reviewer: ```suggestion
## Add-on configuration example
```
